staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go

```diff
@@ -100,7 +100,7 @@ func NewCmdRolloutHistory(f cmdutil.Factory, streams genericclioptions.IOStreams
 	return cmd
 }
 
-// Complete completes al the required options
+// Complete completes all the required options
```

Comment:
The same typo is here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_undo.go#L104 please fix this too.

Reply:
Aye